www/app/outline_server.ts

```diff
@@ -98,18 +83,4 @@ export class OutlineServer implements PersistentServer {
   checkReachable(): Promise<boolean> {
     return this.connection.isReachable();
   }
-
-  private start(): Promise<void> {
```

Comment:
Is this not used?

Reply:
It was only used by `connect()` - there was no obvious reason for it to be factored out like that.